.gitignore

```diff
@@ -2,3 +2,4 @@
 __pycache__
 .DS_Store
 /settings.json
+.vscode
```

Comment:
Same as below. I don’t think we should do a wholesale exclude of the vscode directory, because it is common practice to include a vscode/settings.json file for shared editor configs.

See: https://stackoverflow.com/a/57749909

Reply:
I can revert it.

Reply:
Yes, please revert.

Reply:
Done